release/fixtures/fcos-releases.json

```diff
@@ -8,6 +8,13 @@
           "checksum": "113aa27efe1bbcf6324af7423f64ef7deb0acbf21b928faec84bf66a60a5c933"
         }
       ],
+      "oci-images": [
+        {
+          "architecture": "x86_64",
```

Comment:
Do we really need this versus just pointing at the manifest listed version?

Reply:
We need to know which architectures are available for building the graph, and we don't want to have to query a registry for it. But I guess we could have it be e.g.

```json
"oci-images": {
  "architectures": [],
  "image": "quay.io/...",
  "digest-ref": "quay.io/..."
}
```

?

I think this would work, and it's definitely tempting. The reason I wrote it this way was (1) to match the semantics of `commits` to make this easier to wire through in the places that need it, and (2) because the release metadata itself already breaks it down into arches. We'd have to either add the manifest listed version to the release metadata as a top-level field alongside `architectures` or I guess have the same manifest listed digest in all the arches, which is awkward.

Not sure, I think it'd be nice but I don't think it matters much and would ideally require reworking the schemas a bit more to do this cleanly. (Also looks like `kubevirt`, the only OCI image we currently have in release metadata, does not currently use the manifest listed version so at least we're consistent with that too.)

Hmm, I guess another minor argument is that the way `plume release update-release-index` works, it's meant to be purely additive (because e.g. you could be running it multiple times, each time adding info for another arch that successfully built for a specific version). Whereas for this, it'd have to edit an existing field because on each run, the manifest listed digest would have changed. This mutation also surfaces to the graph, where nodes in the graph could have payloads that change values which I think won't break things but definitely seems odd.